src/server/gc_worker/mod.rs

```diff
@@ -1597,6 +1614,52 @@ impl<E: Engine> GcWorker<E> {
             .into_stream()
             .flatten()
     }
+
+    pub fn start_collecting(
+        &self,
+        max_ts: TimeStamp,
+        callback: LockCollectorCallback<()>,
+    ) -> Result<()> {
+        self.applied_lock_collector
+            .as_ref()
+            .ok_or_else(|| box_err!("applied_lock_collector not supported"))
+            .and_then(move |c| c.start_collecting(max_ts, callback))
+    }
+
+    pub fn get_collected_locks(
+        &self,
+        max_ts: TimeStamp,
+        callback: LockCollectorCallback<(Vec<LockInfo>, bool)>,
+    ) -> Result<()> {
+        self.applied_lock_collector
+            .as_ref()
+            .ok_or_else(|| box_err!("applied_lock_collector not supported"))
+            .and_then(move |c| c.get_collected_locks(max_ts, callback))
+    }
+
+    pub fn stop_collecting(
+        &self,
+        max_ts: TimeStamp,
+        callback: LockCollectorCallback<()>,
+    ) -> Result<()> {
+        self.applied_lock_collector
+            .as_ref()
+            .ok_or_else(|| box_err!("applied_lock_collector not supported"))
+            .and_then(move |c| c.stop_collecting(max_ts, callback))
+    }
+
+    pub fn change_io_limit(&self, limit: i64) -> Result<()> {
+        let mut limiter = self.limiter.lock().unwrap();
+        if limit == 0 {
+            limiter.take();
+        } else {
+            limiter
+                .get_or_insert_with(|| RocksIOLimiter::new(limit))
+                .set_bytes_per_second(limit as i64);
+        }
+        info!("GC io limit changed"; "max_write_bytes_per_sec" => limit);
+        Ok(())
+    }
```

Comment:
We don't need it any more?

Reply:
Oh, sorry, I failed to resolve merge conflicts...